site/controllers/city.js

```diff
@@ -0,0 +1,24 @@
+const City = require("../models/city");
+
+exports.getCity = async (req, res, next) => {
+  const city = await City.findAll();
+  console.log(req.city);
+  res.send(city);
+};
+
+exports.getAddCity = (req, res, next) => {
+  res.render("addCity.jsx");
```

Comment:
since we're working with react as front end, you'll need to used res.json() to send jsong data as a response